src/platform/EFR32/CHIPDevicePlatformConfig.h

```diff
@@ -36,6 +36,8 @@
 #define CHIP_DEVICE_CONFIG_ENABLE_THREAD 1
 #define CHIP_DEVICE_CONFIG_ENABLE_MDNS 1
 #define CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT 1
+#define CHIP_DEVICE_CONFIG_ENABLE_EXTENDED_DISCOVERY 1
+#define CHIP_DEVICE_CONFIG_ENABLE_UNPROVISIONED_MDNS 1
```

Comment:
Why is this called MDNS when it's DNS-SD ? We should rename this `CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONABLE_DISCOVERY`. On thread devices, it should use SRP.

Reply:
done